packages/teleport/src/DesktopSession/TdpClientCanvas/useTdpClientCanvas.tsx

```diff
@@ -141,6 +141,16 @@ export default function useTdpClientCanvas(props: Props) {
   };
 }
 
+// Calculates the size (in pixels) of the display.
+// Since we want to maximize the display size for the user, this is simply
+// the full width of the screen and the full height sans top bar.
+function getDisplaySize(): { width: number; height: number } {
```

Comment:
lets rely on type inference here and remove this return type definition (this is an internal method so it's OK)